plot/assets/chart.html

```diff
@@ -108,6 +116,12 @@
           >
         </li>
         {{end}}
+        <li style="float:right">
```

Comment:
`li:last-child` has the float attribute, it is necessary here?

Reply:
Oh i see, i will move this one to style block and i will add more test to pass coverage kub 😆